src/basic/missing_syscall.h

```diff
@@ -329,7 +329,7 @@ static inline ssize_t copy_file_range(int fd_in, loff_t *off_in,
 #    elif defined __x86_64__
 #      define __NR_bpf 321
 #    elif defined __aarch64__
-#      define __NR_bpf 386
+#      define __NR_bpf 280
```

Comment:
hmm? your other patch patches this again?

Please make sure to submit "perfect" PRs, i.e. where each commit reflects a logical step, but not a historical one. i.e. where eahc commit makes sense on its own, and no fix-up commits are included in a PR that correct what an earlier commit in the same PR did wrong...